src/citrine/informatics/predictors/chemical_formula_featurizer.py

```diff
@@ -0,0 +1,162 @@
+    - "Elastic Poisson Ratio"
+    - "DFT energy density"
+    - "Interatomic distance"
+    - "Ionization Affinity Ratio"
+    - "Ratio of Electron Affinity to Electronegativity"
+    - "Trouton's Ratio"
+    - "Miracle Ratio"
+    - "DFT volume ratio"
+    - "Mulliken electronegativity"
+    - "Modulii sum"
+    - "Zunger Pseudopotential radius ratio"
+    - "BCC Efficiency"
+    - "Non-dimensional heat of fusion"
+    - "Non-dimensional band gap"
+    - "Conduction ionization energy"
+    - "Valence electron density"
+    - "Non-dimensional work function"
+    - "Shear Modulus Melting Temp Product"
+
+    The following features are not weighted means. Their values do not depend on ``powers``.
+
+    - "Maximum electronegativity difference"
+    - "Maximum radius difference"
+    - "Maximum radius ratio"
+    - "Min atomic radius plus max electronegativity difference"
+    - "Number of elements"
+    - "Minimum atomic fraction"
+    - "Maximum atomic fraction"
+    - "Minimum weight fraction"
+    - "Maximum weight fraction"
+    - "Formula weight"
+
+    Parameters
+    ----------
+    input_descriptor: ChemicalFormulaDescriptor
+        the descriptor to featurize
+    features: Optional[List[str]]
+        The list of features to compute, either by name or by group alias. Default is "standard."
+    excludes: Optional[List[str]]
+        The list of features to exclude, either by name or by group alias. Default is none.
+        The final set of features generated by the predictor is set(features) - set(excludes).
+    powers: Optional[List[int]]
+        The list of powers to use when computing generalized weighted means of element properties.
+        p=1 corresponds to the ordinary mean, p=2 is the root mean square, etc.
+
+    """
+
+    input_descriptor = _properties.Object(Descriptor, 'config.input')
+    features = _properties.List(_properties.String, 'config.features')
+    excludes = _properties.List(_properties.String, 'config.excludes')
+    powers = _properties.List(_properties.Integer, 'config.powers')
+
+    typ = _properties.String('config.type', default='ChemicalFormulaFeaturizer',
+                             deserializable=False)
+
+    # NOTE: These could go here or in _post_dump - it's unclear which is better right now
+    module_type = _properties.String('module_type', default='PREDICTOR')
+
+    def __init__(self,
+                 name: str,
+                 description: str,
+                 input_descriptor: ChemicalFormulaDescriptor,
+                 features: Optional[List[str]] = None,
+                 excludes: Optional[List[str]] = None,
+                 powers: Optional[List[int]] = None):
+        self.name = name
+        self.description = description
+        self.input_descriptor = input_descriptor
+        self.features = features if features is not None else ["standard"]
+        self.excludes = excludes if excludes is not None else []
+        self.powers = powers if powers is not None else [2]
```

Comment:
I think we actually want the default to be 1; That's what the SimpleML predictor has been doing.
```suggestion
        self.powers = powers if powers is not None else [1]
```